dev/com.ibm.ws.logging_test/test/com/ibm/ws/logging/internal/impl/BaseTraceFormatterTest.java

```diff
@@ -79,82 +78,160 @@ public void tearDown() {
-        testLoggerEnteringParam("] 0000000b source        >  method ENTRY aaa",
-                                "] 0000000b id=         source                                                       > method ENTRY aaa",
-                                "] 0000000b  > UOW= source=source method=method thread=[Test worker]"
-                                                + LoggingConstants.nl + "          ENTRY aaa"),
-        testLoggerEnteringParams("] 0000000b source        >  method ENTRY aaa bbb ccc",
-                                 "] 0000000b id=         source                                                       > method ENTRY aaa bbb ccc",
-                                 "] 0000000b  > UOW= source=source method=method thread=[Test worker]"
-                                                 + LoggingConstants.nl + "          ENTRY aaa bbb ccc"),
-        testTrExit("] 0000000b source        <  method " + BaseTraceFormatter.EXIT,
-                   "] 0000000b id=         source                                                       < method " + BaseTraceFormatter.EXIT,
-                   "] 0000000b  < UOW= source=source method=method thread=[Test worker]"
-                                   + LoggingConstants.nl + "          " + BaseTraceFormatter.EXIT),
-        testTrExitWithEntryParam("] 0000000b source        <  method " + BaseTraceFormatter.EXIT + " " +
-                                 BaseTraceFormatter.nlBasicPadding + "MapEntry",
-                                 "] 0000000b id=         source                                                       < method " + BaseTraceFormatter.EXIT + " " +
-                                                 BaseTraceFormatter.nlEnhancedPadding + "MapEntry",
-                                 "] 0000000b  < UOW= source=source method=method thread=[Test worker]"
-                                                 + LoggingConstants.nl + "          " + BaseTraceFormatter.EXIT + " "
-                                                 + BaseTraceFormatter.nlAdvancedPadding + "MapEntry"),
-        testLoggerExiting("] 0000000b source        <  method RETURN",
-                          "] 0000000b id=         source                                                       < method RETURN",
-                          "] 0000000b  < UOW= source=source method=method thread=[Test worker]"
-                                          + LoggingConstants.nl + "          RETURN"),
-        testLoggerExitingParam("] 0000000b source        <  method RETURN aaa",
-                               "] 0000000b id=         source                                                       < method RETURN aaa",
-                               "] 0000000b  < UOW= source=source method=method thread=[Test worker]"
-                                               + LoggingConstants.nl + "          RETURN aaa");
+        testNullAll("] " + DataFormatHelper.getThreadId()
```

Comment:
You can use `threadId` instead of `DataFormatHelper.getThreadId()` as `threadId` has been set in line 54.